app/assets/javascripts/dashboard.js

```diff
@@ -1,3 +1,25 @@
+$(document).ready(function() {
```

Comment:
Hi, is the modal code here still needed? It seems like the alert box approach may not require the modal code. Thanks!!

Reply:
Yes, this code is redundant now .Thanks for pointing and I will take care of these types of mistakes for future contributions.